packages/main/src/RPA/Browser/Selenium.py

```diff
@@ -1949,3 +1950,19 @@ def _send_command_and_get_result(self, cmd, params):
         response = self.driver.command_executor._request("POST", url, body)
 
         return response.get("value")
+
+    @keyword
+    def go_to(self, url: str, autoformat=True):
+        """Navigates the current browser window to the provided ``url``.
+
+        If the ``autoformat`` argument is enabled (default), the
+        library adds an ``https://`` prefix if no protocol is given.
+        """
+        if autoformat:
+            url = str(url).strip()
+            if not re.match(r"^\w+:", url):
```

Comment:
While this indeed works, wouldn't be safer to use `urlsplit/urlparse` from `urllib.parse` like so?
```python
>>> urllib.parse.urlsplit("https://www.google.com/path/to?q=1")
SplitResult(scheme='https', netloc='www.google.com', path='/path/to', query='q=1', fragment='')
>>> urllib.parse.urlsplit("about:config").scheme
'about'
```